examples/text-analysis-wordclouds_R-Binder/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# This Dockerfile is based on the rocker/binder example Dockerfile from https://github.com/rocker-org/binder/
+# We use 3.6.0 because it is a recent version of R that has a fixed MRAN date in the Rocker image.
+# The used MRAN URL is: https://mran.microsoft.com/snapshot/2019-07-05
+FROM rocker/binder:3.6.0
+
+## Declares build arguments
+ARG NB_USER
+ARG NB_UID
+
+# Install system dependency for pdftools
+USER root
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+    libpoppler-cpp-dev \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/
+
+## Run install.R script, which installs required R packages and LaTeX packages via tinytex
+COPY install.R ${HOME}
+RUN R --quiet -f install.R
+
+## Copies all repo files into the Docker Container
```

Comment:
Why is the user root directive done twice, and are we sure HOME is defined? Also, same comment about porting to Singularity.

Reply:
We are sure because of the base image that `HOME` is defined.

Double `USER` statement needs to be fixed.